install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -12,262 +12,728 @@ toc::[]
 
 == Overview
 
-This topic provides a general guide on pod security context as it relates to
-volume security. For information on pod security context in general, please see
-link:../../admin_guide/manage_scc.html[Managing Security Context Constraints].
+This topic provides a general guide on pod security as it relates to volume
+security. For information on pod-level security in general, see
+link:../../admin_guide/manage_scc.html[Managing Security Context Constraints
+(SCC)] and the
+link:../../architecture/additional_concepts/authorization.html#security-context-constraints[Security
+Context Constraint] concept topic. For information on the OpenShift persistent
+volume (PV) framework in general, see the
+link:../../architecture/additional_concepts/storage.html[Persistent Storage]
+concept topic.
+
+Accessing persistent storage requires coordination between the cluster and/or
+storage administrator and the end developer. The cluster administrator creates
+PVs, which abstract the underlying physical storage. The developer creates pods
+and, optionally, PVCs, which bind to PVs, based on matching criteria, such as
+capacity.
+
+Multiple persistent volume claims (PVCs) within the same project can bind to the
+same PV. However, once a PVC binds to a PV, that PV cannot be bound by a claim
+outside of the first claim's project. If the underlying storage needs to be
+accessed by multiple projects, then each project needs its own PV, which can
+point to the same physical storage. In this sense, a bound PV is tied to a
+project. For a detailed PV and PVC example, see the guide for
+https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and
+MySQL using NFS].
+
+For the cluster administrator, granting pods access to PVs involves:
+
+- knowing the group ID and/or user ID assigned to the actual storage,
+- understanding SELinux considerations, and
+- ensuring that these IDs are allowed in the range of legal IDs defined for the
+project and/or the SCC that matches the requirements of the pod.
+
```

Comment:
(again, for translation) Better to restructure as a complete sentence as intro, and complete sentences as the points.